sampler/README.md

```diff
@@ -471,7 +473,7 @@ $$
 p(\mathbf{x}; \mathbf{p}) = \frac{\Gamma((\sum_{i=1}^k x_i)+1)}{\Gamma(x_1+1)\cdots{}\Gamma(x_k+1)}p_1^{x_1}\cdots{}p_k^{x_k},
 $$
 
-#### Working in "log-space"
+#### Working in "Log-Space"
```

Comment:
I'm not sure of your use of "log-space" here. You seem to be using it to mean "working with logs", but when I looked it up, all the references seemed to be to " complexity class containing decision problems ... logarithmic amount of memory space" (Wikipedia). Will your usage be familiar to our audience? (Although even if it isn't, I think you have explained it clearly enough at the start of the section.)